pom.xml

```diff
@@ -113,98 +36,15 @@
 					<artifactId>maven-compiler-plugin</artifactId>
 					<version>3.2</version>
 					<configuration>
-						<source>1.7</source>
-						<target>1.7</target>
+						<source>1.5</source>
```

Comment:
This was indeed required at some time, and is now deprecated legacy. Will update it to 1.7.